petstore/clients/javascript/package.json

```diff
@@ -0,0 +1,280 @@
+{
+  "name": "@unbranded/petstore",
+  "version": "1.0.0-beta.1",
+  "description": "A generated SDK for PetStoreClient.",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "sideEffects": false,
+  "autoPublish": false,
+  "tshy": {
+    "exports": {
+      "./package.json": "./package.json",
+      ".": "./src/index.ts",
+      "./models": "./src/models/index.ts",
+      "./api": "./src/api/index.ts",
+      "./api/pets": "./src/api/pets/index.ts",
+      "./api/petCheckups": "./src/api/petCheckups/index.ts",
+      "./api/petInsurance": "./src/api/petInsurance/index.ts",
+      "./api/toys": "./src/api/toys/index.ts",
+      "./api/toyInsurance": "./src/api/toyInsurance/index.ts",
+      "./api/checkups": "./src/api/checkups/index.ts",
+      "./api/owners": "./src/api/owners/index.ts",
+      "./api/ownerCheckups": "./src/api/ownerCheckups/index.ts",
+      "./api/ownerInsurance": "./src/api/ownerInsurance/index.ts"
+    },
+    "dialects": [
+      "esm",
+      "commonjs"
+    ],
+    "esmDialects": [
+      "browser",
+      "react-native"
+    ],
+    "selfLink": false
+  },
+  "type": "module",
+  "scripts": {
+    "clean": "rimraf --glob dist dist-browser dist-esm test-dist temp types *.tgz *.log",
+    "extract-api": "rimraf review && mkdirp ./review && api-extractor run --local",
+    "pack": "npm pack 2>&1",
+    "lint": "eslint package.json api-extractor.json src",
+    "lint:fix": "eslint package.json api-extractor.json src --fix --fix-type [problem,suggestion]",
+    "build": "npm run clean && tshy && npm run extract-api"
+  },
+  "devDependencies": {
+    "@microsoft/api-extractor": "^7.40.3",
+    "@types/node": "^18.0.0",
+    "eslint": "^9.9.0",
+    "typescript": "~5.6.2",
+    "tshy": "^2.0.0",
+    "rimraf": "^5.0.5",
+    "mkdirp": "^3.0.1"
+  },
+  "dependencies": {
+    "tslib": "^2.6.2",
+    "@typespec/ts-http-runtime": "1.0.0-alpha.20240314.2"
```

Comment:
@v-jiaodi could you check if this is the latest version? if not pls update the version in codegen.